grails-app/assets/javascripts/managementUnitReport.js

```diff
@@ -0,0 +1,36 @@
+function ManagementUnitReportSelectorViewModel() {
```

Comment:
Pass a config parameter to the view model constructor containing the two URLs used in this model.  This helps by defining the dependencies explicitly rather than relying on the fcConfig global